stackme-kit/core/core.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BasicCore } from './core.composition';
+
+it('should render with the correct text', () => {
+  const { getByText } = render(<BasicCore />);
+  const rendered = getByText('hello from Core');
+  expect(rendered).toBeTruthy();
+});
```

Comment:
wrap this one with describe for the better test suite
`describe('Basic core component unit test'){
--- your code right here ---
}
`